mimesis/decorators.py

```diff
@@ -1,4 +1,6 @@
-import functools
+from typing import Any, Callable, TypeVar, Generator, Iterable, cast
+U = TypeVar('U', covariant=True)
```

Comment:
What is mean `U`?

Reply:
@lk-geimfari It's a type variable. I've already updated this too in a commit i'll make later with fixes as to your other comments here.